config/tests/testgrids/config_test.go

```diff
@@ -85,8 +91,33 @@ var prowConfig *prow_config.Config
 func TestMain(m *testing.M) {
 	flag.Parse()
 	if *protoPath == "" {
-		fmt.Println("--config must be set")
-		os.Exit(1)
+		// Read from testgrid
+		tmpDir, err := ioutil.TempDir("", "testgrid-config-test")
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		defer os.RemoveAll(tmpDir)
+		tmpFile := path.Join(tmpDir, "test-proto")
+
+		opt := options.Options{
+			Inputs: []string{"../../testgrids"},
```

Comment:
Let's make this a flag with this as the default

Reply:
done, but not as clean as the others